ecs.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Blumilk\Codestyle\Config;
+use Blumilk\Codestyle\Configuration\Defaults\Paths;
+
+$config = new Config(
+    paths: new Paths("backend")
```

Comment:
What will be in the `backend` dir?

Reply:
We are splitting development into backend and frontend. In backend, there will be php api for generating static webpage files from data in resources folder. So php classes will be contained in that directory.